src/library/library.cpp

```diff
@@ -369,6 +455,94 @@ QStringList Library::getDirs() {
     return m_pTrackCollection->getDirectoryDAO().getDirs();
 }
 
+void Library::paneCollapsed(int paneId) {
```

Comment:
Yes keep it in. The multipane feature will be the next step and it most likely needs this code.